multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/api/impl/FilesApiServiceImpl.java

```diff
@@ -424,13 +433,22 @@ private HttpResponse<InputStream> callRemoteEndpointWithRetry(HttpClient client,
                                                                UriUtil.stripUserInfo(decodedUrl));
                     throw new SLException(errorMessage);
                 }
-                throw new SLException(MessageFormat.format(Messages.ERROR_FROM_REMOTE_MTAR_ENDPOINT, request.uri(), response.statusCode(),
-                                                           error));
+                throw new SLException(MessageFormat.format(Messages.ERROR_FROM_REMOTE_MTAR_ENDPOINT, getUriFromAtSign(request.uri()),
```

Comment:
Can we add a unit test for this to ensure that credentials would not be present in the logs?